distribution/src/main/packaging/scripts/postinst

```diff
@@ -61,9 +61,9 @@ if [ "x$IS_UPGRADE" != "xtrue" ]; then
 
     elif command -v chkconfig >/dev/null; then
```

Comment:
This line is going to fail also, since chkconfig won't be in the path, perhaps what we need is the `-p` option:

```
command [-pVv] command [arg ...]
              Run  command with args suppressing the normal shell function lookup. Only builtin commands or commands found in the PATH are executed.  If the -p option is given, the search for command is per‐
              formed using a default value for PATH that is guaranteed to find all of the standard utilities.
```

This should be tested before merging though